test/com/linkedin/drelephant/BackfillTest.java

```diff
@@ -232,7 +232,7 @@ public void run() {
         try {
           runnerThread.join();
         } catch (InterruptedException e) {
-          e.printStackTrace();
+          // Ignore the exception.
```

Comment:
So this is the last step in the test to explicitly kill the thread. And the exception has been caught primarily because Thread#join explicitly throws a checked exception. 
If the test has to fail, it would have failed already in previous asserts. If by any chance thread is interrupted (probably would only happen if we press Ctrl+C), this would have spit out a stack trace, which I just felt that can confuse the user. If the code hits this line, the test has already passed.
I myself had added printStackTrace in the previous PR, but removed it here, because I felt it was confusing.
If you think otherwise, I can change it back to previous state as I do not necessarily have strong opinions about this change.